students/Wayne/Assignments/Lesson2/PrinterGridExercise.py

```diff
@@ -0,0 +1,91 @@
+#print grid with single variable
+
+# declaring grid variables
+
+plus = '+'
+
+endplus = '+'
+
+minus = ' - '
+
+line = '|'
+
+endline = '|'
+
+space = '   '
+
+#Building function to build a single variable print function 
+
+def print_grid(n):
+    count = n
+    centerrow = ((n)//2)+1
+    center = ((n)//2)-1
+    row = (plus + (minus*center) + plus + (minus*center) +plus)
+    column = (line + (space * center) + line + (space * center) +line)
+    
+    while count >= 1:
+        if count ==1:
```

Comment:
While I think the for loop approach (see an example in the solutions folder on github) is a little cleaner, I really appreciate the logic behind this. 
A couple of comments:
- -= is a little cleaner here
- if you have the same result, (i.e. "print(row)" under different elifs, it's better to combine them for a number of reasons. It's more readable and it's better for editing later if you have to change, for example, what they're printing. 
```
if count == 1 or count == n or count == center_row:
    print(row)
else:
    print(column)
count -= 1